### PR TITLE
Stack headless

### PR DESCRIPTION
**Pull request by FynnBe**
complements pull request https://github.com/ilastik/ilastik/pull/1685

### CODE REVIEW

File: lazyflow/operators/ioOperators/opInputDataReader.py [hunk 1]
```diff
@@ -67,7 +65,9 @@
 import re
 import logging
 
-from lazyflow.utility.io_util.multiprocessHdf5File import MultiProcessHdf5File
+from lazyflow.utility.io_util.multiprocessHdf5File import \
+    MultiProcessHdf5File
```
Comment (k-dominik) [inline lazyflow/operators/ioOperators/opInputDataReader.py]: I'm puzzled about this one, what is your line length limit? I think we have agreed to 120.

Comment (k-dominik) [reply lazyflow/operators/ioOperators/opInputDataReader.py]: this was just an exemplary comment, there are a lot of these changes

**Comment by FynnBe** [comment]
I removed the PEP8 commit, which only included limiting the line length to 80.

**Comment by k-dominik** [comment]
Thank you for removing the unnecessary pep8 stuff.

Given your latest additions a test would really help. It would be really good to add something to `testOpInputDataReader` similar to `TestOpInputDataReader.test_h5_stack_single_file` for each stacking axis in `ztc`, for a bunch of 2D `.png`s or something else that is opened with `OpStackLoader`

File: lazyflow/operators/ioOperators/opInputDataReader.py [hunk 2]
```diff
@@ -324,6 +324,7 @@ def _attemptOpenAsTiffStack(self, filePath):
     def _attemptOpenAsStack(self, filePath):
         if '*' in filePath or os.path.pathsep in filePath:
             stackReader = OpStackLoader(parent=self)
+            stackReader.SequenceAxis.connect(self.SequenceAxis)
```
Comment (k-dominik) [inline lazyflow/operators/ioOperators/opInputDataReader.py]: btw, this should make it work with pngs

**Comment by k-dominik** [comment]
cool!

**Pull request merged**